src/pages/api-showcase.tsx

```
@@ -0,0 +1,459 @@
+  description: string;
+  loader: (
+    good: SafeApiClient<AlcoholZincApi<unknown>>,
+    bad: SafeApiClient<AlcoholZincApi<unknown>>,
+  ) => Promise<Result<any, Problem>>;
+  icon: React.ReactNode;
+}
+
+interface SectionData {
+  sectionId: string;
+  status: 'idle' | 'loading' | 'success' | 'error';
+  data?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)